app/views/categories/show.html.erb

```diff
@@ -10,6 +10,15 @@
   <%= @category.description %>
 </p>
 
+<p>
+  <strong>Carriers:</strong>
```

Comment:
@abronkema if category doesn't have carriers do we want to show some message ex: `No carriers`...atm it will just be displayed as `Carriers:`